cips/cip-x.md

```diff
@@ -0,0 +1,77 @@
+| cip | XX (assigned by Editors) |
+| - | - |
+| title | Header Pruning for Light Nodes |
+| description | Mechanism to retain a fixed range of headers instead of whole history |
+| author | Hlib Kanunnikov ([@Wondertan](https://github.com/Wondertan)) |
+| discussions-to | <https://forum.celestia.org/t/cip-header-pruning-for-lns/1958> |
+| status | Draft |
+| type | Standards Track |
+| category | Data Availability |
+| created | 2025-03-21 |
+
+## Abstract
+
+Currently, every data availability (DA) node type synchronizes all historical headers starting from genesis (or other statically configured
+historical header) until the subjectively initialized head of the chain. We change that by adding a way to sync a
+constant size range of headers instead of the whole history.
+
+## Motivation
+
+Light nodes (LNs) store the whole history of ExtendedHeaders in their entirety, including ValidatorSet, Commit and data availability header (DAH). The motivation
+is to decrease LN storage requirements by retaining only a configurable window of recent headers.
```

Comment:
How should one reason about what an appropriate configuration for this should be?

Reply:
i see this is basically alluded to in the parameter section but it would be nice if it was clear up front in the motivation